### DISCUSSION TITLE
Can't install cflib

### DISCUSSION THREAD

**Discussion by HumbledBezalel** [bitcraze/discussions#477] [Q&A]
What once I get to this part: 
<img width="1075" alt="image" src="https://user-images.githubusercontent.com/98075546/208271215-2e2fa9cc-9569-4be8-98b7-d058bdd404b6.png">
 I get errors, I tried with pip and python3:
<img width="660" alt="image" src="https://user-images.githubusercontent.com/98075546/208271243-0f491c39-7f41-425e-bd85-bfbec8516b10.png">

I have a Macbook M1. What am I doing wrong?

**Comment by ataffanel** [comment]
Hi,

The fact that you do not have ```pip``` in your path (ie. it yields 'Command not found') is not good. Have you followed the [Mac M1 part of the documentation](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/installation/install/#apple-m1)?

On a Mac it is very important to make sure you are using the right installation of python (as unfortunatly the apple-provided installation will not work for most purpose ...). And in the time of writing of the documentation a X86 version of python was required, it should install in ```/usr/local/```. In your case you can try to run directly ```/usr/local/bin/pip install -e .``` to make sure you are using the brew-provided x86 Python.
